dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -246,6 +248,59 @@ def test_add_profilers(self, *mocks):
         self.assertEqual("multiple files", merged_profile.encoding)
         self.assertEqual("multiple files", merged_profile.file_type)
 
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder." "ColumnPrimitiveTypeProfileCompiler"
+    )
+    @mock.patch("dataprofiler.profilers.profile_builder." "ColumnStatsProfileCompiler")
+    @mock.patch("dataprofiler.profilers.profile_builder." "ColumnDataLabelerCompiler")
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder.DataLabeler", spec=StructuredDataLabeler
+    )
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder."
+        "StructuredProfiler._update_correlation"
+    )
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder."
+        "StructuredProfiler._merge_correlation"
+    )
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder." "StructuredProfiler._update_chi2"
+    )
+    def test_add_profiles_null_count_not_enabled(self, *mocks):
+        profiler1_options_null_count = ProfilerOptions()
```

Comment:
instead of `profiler1_options_null_count` or `profiler2_options_null_count`. I'd suggest naming them for clairity wrt to their differences to make the test more readable.

e.g. `profiler1_options_null_count` -> `profiler_options_null_disabled`

similarly, `profiler1` -> `profiler_w_disabled_null_count`

Reply:
also these tests should be moved down to the `TestStructuredProfilerRowStatistics`.